oshmem/proc/proc.h

```diff
@@ -35,30 +37,15 @@ struct oshmem_group_t;
 
 #define OSHMEM_PE_INVALID   (-1)
 
-/**
- * Remote Open SHMEM process structure
- *
- * Remote Open SHMEM process structure.  Each process contains exactly
- * one oshmem_proc_t structure for each remote process it knows about.
- */
-struct oshmem_proc_t {
-    opal_proc_t                    super;
-    /* endpoint data */
-    void *proc_endpoints[OMPI_PROC_ENDPOINT_TAG_MAX];
-    /*
-     * All transport channels are globally ordered.
-     * pe(s) can talk to each other via subset of transports
-     * these holds indexes of each transport into global array
-     *  proc -> id, where id can be btl id in yoda or mxm ptl id
-     *  in ikrit
-     *  spml is supposed to fill this during add_procs()
-     **/
-    int                             num_transports;
-    char                           *transport_ids;
+struct oshmem_proc_data_t {
+    int num_transports;
+    char * transport_ids;
```

Comment:
That doesn't matter.

The only ABI we guarantee is the user-facing MPI and OSHMEM APIs.

We specifically do *not* guarantee internal ABI between different releases of Open MPI.  Specifically, we still require that if you use Open MPI vX.Y.Z with one process, you must use Open MPI vX.Y.Z in *all* processes *and* the runtime.